test/integration/package_int_test.go

```diff
@@ -501,6 +470,13 @@ func (suite *PackageIntegrationTestSuite) TestJSON() {
 	cp.ExpectExitCode(0)
 	AssertValidJSON(suite.T(), cp)
 
+	cp = ts.SpawnWithOpts(
+		e2e.OptArgs("checkout", "ActiveState-CLI/Packages-Perl", "."),
+		e2e.OptAppendEnv("ACTIVESTATE_CLI_DISABLE_RUNTIME=false"),
```

Comment:
I think we can remove this because the runtime is being sourced in the next operation.

Reply:
Removing this line causes the tests to panic.